### DISCUSSION TITLE
2024 Porsche Boxster Issue - MHI2_ER_POG11_P5250

### DISCUSSION THREAD

**Discussion by Phelpsator** [Mr-MIBonk/M.I.B._More-Incredible-Bash#461] [Q&A]
Greetings folks. I'm the happy new owner of a Porsche 718 Boxster MY2024 :-) 

When trying to install MIB from the developper menu, I get an error that the file metainfo2.txt is invalid. My MHI version is MHI2_ER_POG11_P5250 and the MU software is 1502. 
Am I doing something wrong, or is this version just too recent and hasn't been incorporated in MIB yet? (I noticed there is a patch for my version, but there doesn't seem to be much inside)...

Thanks for any insights/help you can provide!

Best regards

**Comment by josbmw** [comment]
New 2024 GTS 4.0 over here. Same MHI version, same issue.
Any solution in sight?

**Reply by darkmrscod** [reply]
Hello

I have the solution. Activation only via SD, no dlink needed. Already did on Rs Spyder 2024

**Reply by josbmw** [reply]
Nice. How do I get it?

**Reply by darkmrscod** [reply]
It me up on WA: https://wa.me/message/BICIF6JYO33ZA1

**Reply by josbmw** [reply]
I don't use Whatsapp, sorry. No Meta software whatsoever ..

**Reply by darkmrscod** [reply]
Do u have an email ?

**Comment by Phelpsator** [comment]
From what I understand, this is caused by a complete change in the HW and removal of the CD unit on the 2024 model. I hope someone figures it out!

**Comment by lukaspour** [comment]
Same issue here. Following

**Reply by darkmrscod** [reply]
Hello

I have the solution. Activation only via SD, no dlink needed. Already did on Rs Spyder 2024

**Comment by HansiNEA** [comment]
Same car, same problem. My MHI is without Navigation, can I activate it also? When the issue is solved.

**Reply by darkmrscod** [reply]
Hello

I have the solution. Activation only via SD, no dlink needed. Already did on Rs Spyder 2024

**Comment by darkmrscod** [comment]
> Greetings folks. I'm the happy new owner of a Porsche 718 Boxster MY2024 :-)
> 
> When trying to install MIB from the developper menu, I get an error that the file metainfo2.txt is invalid. My MHI version is MHI2_ER_POG11_P5250 and the MU software is 1502. Am I doing something wrong, or is this version just too recent and hasn't been incorporated in MIB yet? (I noticed there is a patch for my version, but there doesn't seem to be much inside)...
> 
> Thanks for any insights/help you can provide!
> 
> Best regards

Hello

I have the solution. Activation only via SD, no dlink needed. Already did on Rs Spyder 2024

**Comment by edgaro28** [comment]
On thoes models h60 metainfo bug dosent work anymore

**Reply by darkmrscod** [reply]
lol
